analysis/tests/src/expected/CompletionFunctionArguments.res.txt

```diff
@@ -25,7 +25,8 @@ Completable: Cexpression CArgument Value[someFn](~isOn)=t
     "kind": 4,
     "tags": [],
     "detail": "bool",
-    "documentation": null
+    "documentation": null,
+    "sortText": "AAA1 true"
```

Comment:
Do we need `AAA`?
Do we need `true`?
What's bigger between `"1"` and `"10`"?

Reply:
Changed to A + the actual name. Should be enough.